terms.html

```diff
@@ -51,9 +51,12 @@
   </dd>
   <dt><dfn data-lt="entities|entity's">entity</dfn></dt>
   <dd>
-A thing with distinct and independent existence, such as a person,
-organization, or device that performs one or more roles in the ecosystem.
-  </dd>
+Any abstract or concrete noun that can be referenced in statements.
```

Comment:
This doesn't seem quite right as it changes the entity from that which is referenced to a "word" (a noun). Suggestion:

```suggestion
Anything that can be referenced in statements as an abstract or concrete noun.
```